.github/workflows/Dockerfile.yml

```diff
@@ -16,7 +16,7 @@ concurrency:
 
 env:
   HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
-  IMAGE: ${{ format('{0}/{1}:{2}', secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io', github.repository, github.event_name == 'pull_request' && format('pr-{0}', github.event.pull_request.number) || 'latest') }}
+  IMAGE: ${{ format('{0}/stellar/stellar-system-test:{1}', secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io', github.event_name == 'pull_request' && format('pr-{0}', github.event.pull_request.number) || 'latest') }}
```

Comment:
The reason the quickstart image, that this was copied from, uses `github.repository` is so that if someone forks it and runs the workflow, it still works, and it publishes to the same namespace as the repository.

It should be fine to publish this to dockerhub as `stellar/system-test`. If we really want a stellar prefix, how about we add that prefix to this repo?

Reply:
yes, I noticed the slight difference from the gh repo name that crept up there, [I'm re-asking ops](https://github.com/stellar/ops/issues/1935#issuecomment-1347697296) on prior ticket to help me out there on rename.

Reply:
updated to use use gh repo name for docker repo name, filed new request to get that repo renamed - https://github.com/stellar/ops/issues/2018, this pr will wait till that happens.